### ISSUE TITLE
Refactor/structure ideas

### ISSUE THREAD

**Issue by lizgzil** [wellcometrust/nutrition-labels#62]
Some refactoring/restructuring ideas:
- [x] Merge aspects of `grant_data_processing.py` and `prodigy_training_data.py` to use original data plus Prodigy tags, minus EPMC and RF data to create training data csvs (due to findings from https://github.com/wellcometrust/nutrition-labels/pull/61/files).
- [x] During training of each model output the test predictions (grant ID - prediction pairs), which means `predict_tech.py` wouldn't need to be run. Consider a good way to output train/test split here too.
- [x] Don't train using grant types? Bit worried that since the training set is so small it might overfit to these.
- [x] Consider cleaning the text more - not sure if they are always as clean as they could be
- [x] Use the Voting classifier from WellcomeML/modify it https://github.com/wellcometrust/nutrition-labels/issues/59

**Comment by lizgzil** [comment]
re. cleaning the text more:

currently in `clean_grants_data`
```
{'Description': '<p style="margin-left: 0px; margin-right: 0px"><strong>This grant</strong> is one with html.</p><p style="margin-left: 0px; margin-right: 0px">&nbsp;</p><ul style="list-style-type: disc"><li>It includes a list,</li><li>with some text.</li></ul>'
```
becomes
```
This grant is one with html.\xa0It includes a list,with some text.
```
(should be a space) - so need to take out the `\xa0` or any other latin-8 encodings like this?

e.g. string = string.replace(u'\xa0', u' ')

**Comment by aCampello** [comment]
I think using grant types is a bit problematic giving the new strategy - they are probably going to change, so using something that only uses text or fields that won't change is preferable?

**Comment by lizgzil** [comment]
@aCampello yes that's a good point
